lib/writer.ts

```diff
@@ -259,11 +267,19 @@ export class ParquetEnvelopeWriter {
   writeBloomFilters() {
     this.rowGroups.forEach(group => {
       group.columns.forEach(column => {
-        const columnName = column.meta_data?.path_in_schema[0];
-        if (!columnName || columnName in this.bloomFilters === false) return;
+        if (!column.meta_data) {
+          return
+        }
+        if (!column.meta_data.path_in_schema.length) {
+          return
+        }
```

Comment:
I believe this is equivalent with optional chaining
```suggestion
        if (!column.meta_data?.path_in_schema.length) {
          return
        }
```